README.MD

```diff
@@ -130,20 +129,22 @@ Other useful resources:
    * The [_Real-Time Rendering_ Ray Tracing Resources Page](http://www.realtimerendering.com/raytracing.html), which includes a free chapter on the basic ideas behind DXR and interactive ray tracing.
 
 ----------------------------------------------------------------------------------------------
-Acknowledgments:
+Acknowledgements:
```

Comment:
FWIW, this change makes it into the spelling that the British prefer, e.g., https://www.grammarly.com/blog/what-is-the-difference-between-acknowledgement-and-acknowledgment/